monai/bundle/scripts.py

```diff
@@ -236,11 +252,17 @@ def download(
         extractall(filepath=filepath, output_dir=bundle_dir_, has_base=True)
     elif source_ == "github":
         if name_ is None:
-            raise ValueError(f"To download from source: Github, `name` must be provided, got {name_}.")
+            raise ValueError("To download from source: github, `name` must be provided, got None.")
         _download_from_github(repo=repo_, download_path=bundle_dir_, filename=name_, progress=progress_)
+    elif source_ == "ngc":
+        if name_ is None:
+            raise ValueError("To download from source: ngc, `name` must be provided, got None.")
+        if version_ is None:
+            raise ValueError("To download from source: ngc, `version` must be provided, got None.")
+        _download_from_ngc(download_path=bundle_dir_, filename=name_, version=version_, progress=progress_)
     else:
         raise NotImplementedError(
-            f"Currently only download from provided URL in `url` or Github is implemented, got source: {source_}."
+            f"Currently only download from provided URL in `url`, from source Github or NGC are implemented, got source: {source_}."
```

Comment:
There is a small inconsistency of word capitalization here: the source must be "github" or "ngc" , but the message says "Github" and "NGC".

Reply:
Better to add the `string.lower()` conversion.

Thanks.